### ISSUE TITLE
Project cannot start if using customize-cra instead of react-app-rewired

### ISSUE THREAD

**Issue by zcmgyu** [arackaf/customize-cra#1]
**My configs:**
```
// package.json
"scripts": {
    "start": "react-app-rewired start",
}
```

```
// config-overrides.js
const { addBabelPlugin } = require('customize-cra');

module.exports = function override(config, env) {
  config = addBabelPlugin(
    ['import', { libraryName: 'antd', libraryDirectory: 'es', style: true }], // change importing css to less
    config,
  );
  return config;
};
```
**Throw below error:**
> Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
>  - configuration should be an object.

**Comment by jgoux** [comment]
The function provided by customize-cra mutate the config object without returning it.


This should work : 
```js
module.exports = function override(config, env) {
  addBabelPlugin(
    ['import', { libraryName: 'antd', libraryDirectory: 'es', style: true }], // change importing css to less
    config,
  );
  return config;
};
```

**Comment by arackaf** [comment]
Methods now return the config object, so the above code should work.  Released in 0.0.6
